formidable/models.py

```diff
@@ -32,6 +32,19 @@ def get_next_field_order(self):
         agg = self.fields.aggregate(models.Max('order'))
         return agg['order__max'] + 1 if agg['order__max'] is not None else 0
 
+    @staticmethod
+    def from_json(json_definition):
+        """
+        Proxy (static) method to create an instance of ``Formidable``
+        more easily with a given ``json_definition``.
+
+        >>> form = Formidable.from_json(json_definition)
+        >>> form.save()
+
+        """
+        from formidable.serializer import FormidableSerializer
```

Comment:
The idea is pretty good, but I think we have to check json is correct first

```python

    serializer = FormidableSerializer(data=json_definition)
    if serializer.is_valid():
         return serializer.save()
    raise ValidationError(serializer.errors)

```

Moreover, becareful about json_definition which actually is contextualize... We have to discuss about it, because this kind of json_definition here has to be a complete form definition. We could pre-process the json_definition in case of contextualize json_definition.... I don't have a good idea right now we have to brainstorm about it.